kobo/rcS

```diff
@@ -45,6 +45,13 @@ if [ -f /mnt/onboard/.kobo/Kobo.tgz -o -f /mnt/onboard/.kobo/KoboRoot.tgz \
     exec /etc/init.d/rcS
 fi
 
+# Figure out our platform.
+PLATFORM=freescale
+if [ `dd if=/dev/mmcblk0 bs=512 skip=1024 count=1 | grep -c "HW CONFIG"` == 1 ]; then
+	CPU=`ntx_hwconfig -s -p /dev/mmcblk0 CPU`
```

Comment:
`ntx_hwconfig` is a tool used to read blob values from linux userspace.

This will return a cpu identifier, such as 507,508, etc.

Some legacy versions might ship a blob older than the `ntx_hwconfig` tool and that will cause the command to throw some errors to `stderr`, so it is a good idea to redirect errors with:

```
CPU=$(ntx_hwconfig -s -p /dev/mmcblk0 CPU 2>/dev/null)